### DISCUSSION TITLE
"Smaller than" responsive prefixes?

### DISCUSSION THREAD

**Discussion by kinsi55** [tailwindlabs/tailwindcss#2411] [Ideas]
Assuming I have a div which needs to be full width on LG and below, but full height on XL and above, currently this needs to be expressed like this as far as I can tell

`w-full xl:w-auto xl:h-full`

This only gets worse the more things need to be a certain way on sizes below X but a different above

Wouldnt it be useful to have "Smaller than" prefixes? Something like this

`bxl:w-full xl:h-full` with b standing for below

**Comment by kinsi55** [comment]
I guess for now I've added them myself with the config file.

```
		screens: {
			sm: "640px",
			md: "768px",
			lg: "1024px",
			xl: "1280px",

			bxl: {max: "1279px"},
			blg: {max: "1023px"},
			bmd: {max: "767px"},
			bsm: {max: "639px"}
		}
```

**Reply by mzaini30** [reply]
Is that work? I'll try later

**Reply by dance2die** [reply]
> Is that work? I'll try later

You can find the documentation here: https://tailwindcss.com/docs/breakpoints#max-width-breakpoints

**Reply by mzaini30** [reply]
Thanks
